inference-engine/src/mkldnn_plugin/mkldnn_node.cpp

```diff
@@ -466,9 +466,9 @@ bool MKLDNNNode::canBeInPlace() const {
             return false;
     }
 
-    auto inShape = getParentEdgeAt(0)->getShape();
-    for (size_t cIdx = 0; cIdx < getChildEdges().size(); cIdx++) {
-        if (getChildEdgeAt(cIdx)->getShape() != inShape) {
+    auto inShape = getInputShapeAtPort(0);
+    for (size_t cIdx = 0; cIdx < getOriginalOutputsNumber(); cIdx++) {
```

Comment:
After the offline discussion it was decided to:
1. In this and similar contexts we should iterate using the actual input/output shapes vectors since the `getOriginalOutputsNumber()` corresponds to the ngraph node and may not be equal to the input/output shapes vectors size due to the internal mkldnn graph transformation.
2. Please add corresponding description to `getOriginalOutputsNumber()` and `getOriginalInputsNumber()` methods.